core/common.py

```diff
@@ -20,32 +17,34 @@
 import discord
 import requests
 import sentry_sdk
-from github import Github
 from botocore.exceptions import ClientError
 from discord import (
     Button,
     ButtonStyle,
     DiscordException,
     SelectOption,
-    ui,
+    ui
 )
 from discord.ext import commands, menus
 from dotenv import load_dotenv
+from github import Github
 from google.cloud import secretmanager
 from google_auth_oauthlib.flow import Flow
 from google.oauth2.credentials import Credentials
 from google.oauth2 import service_account
 from oauth2client.service_account import ServiceAccountCredentials
+
 from core import database
 
+
 load_dotenv()
 
-# module variables
+# Module Variables
 coroutineType = Callable[[Any, Any], Awaitable[Any]]
 g = Github(os.getenv("GH_TOKEN"))
 
 
-class ConfigcatClient:
+class ConfigCatClient:
```

Comment:
The change to this class name did not get reflected everywhere, as indicated by this [failed travis build](https://app.travis-ci.com/github/School-Simplified/Timmy-SchoolSimplified/jobs/568014227).

Reply:
i will change these tmrw, but ill double check with puncher since he made many changes